core/base.py

```diff
@@ -54,16 +54,63 @@ class ShareLinksMixin(models.Model):
     cached_last_updated = models.DateTimeField(blank=True, null=True)
 
     def _get_facebook_count(self):
+        # example_json_response = {
+        #    "https://opencanada.org/features/five-issues-should-decide-future-internet/": {
+        #       "og_object": {
+        #          "id": "1330184950343806",
+        #          "description": "With a new report on online governance out, we look at the\nconsiderations...
+        #          "title": "Five issues that should decide the future of the Internet",
+        #          "type": "article",
+        #          "updated_time": "2016-08-23T03:20:45+0000"
+        #       },
+        #       "share": {
+        #          "comment_count": 0,
+        #          "share_count": 87
+        #       },
+        #       "id": "https://opencanada.org/features/five-issues-should-decide-future-internet/"
+        #    },
+        #    "http://opencanada.org/features/five-issues-should-decide-future-internet/": {
+        #       "og_object": {
+        #          "id": "1330184950343806",
+        #          "description": "With a new report on online governance out, we look at the\nconsiderations...
+        #          "title": "Five issues that should decide the future of the Internet",
+        #          "type": "article",
+        #          "updated_time": "2016-08-23T03:20:45+0000"
+        #       },
+        #       "share": {
+        #          "comment_count": 0,
+        #          "share_count": 87
+        #       },
+        #       "id": "http://opencanada.org/features/five-issues-should-decide-future-internet/"
+        #    }
+        # }
         try:
             url = 'https://graph.facebook.com/?ids=https://opencanada.org{0},http://opencanada.org{0}'.format(self.url)
             response = requests.get(url, timeout=5)
             j = response.json()
             total_shares = 0
             for key, values in j.iteritems():
-                total_shares += values.get('shares', 0)
+                share_json = values.get('share', {})
+                facebook_share_count = share_json.get('share_count', 0)
+                if facebook_share_count > total_shares:
+                    total_shares = facebook_share_count
             return total_shares
+        except ValueError:
```

Comment:
This exception should only wrap `j = response.json()`, so that other values error don't get slience.